test/recipes/99-test_fuzz_asn1_0.t

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env perl
+# Copyright 2016-2020 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use warnings;
+
+use OpenSSL::Test qw/:DEFAULT srctop_file/;
+use OpenSSL::Test::Utils;
+
+my $fuzzer = "asn1";
+my $corpus_group = "0";
+setup("test_fuzz_${fuzzer}_${corpus_group}");
+
+plan tests => scalar 2; # one more due to below require_ok(...)
```

Comment:
Oops...  `scalar` is completely unnecessary here...  `2` is already a scalar :wink:

(this applies to all similar lines, of course)

Perl lesson: `scalar @foo` turns the array to a scalar...  in that particular case, into the number of elements

Reply:
Also, the comment is really not necessary...  the two lines that follow are the two tests, it's really very obvious.

Reply:
Good point! fixup commits pushed